data/json/recipes/other/containers.json

```diff
@@ -113,6 +113,24 @@
       [ [ "cordage_short", 3, "LIST" ], [ "filament", 150, "LIST" ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "canteen_wood",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_CONTAINERS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "30 m",
+    "autolearn": true,
+    "using": [ [ "adhesive", 1 ], [ "waterproofing", 1 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "SAW_W", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
+    "components": [
+      [ [ "stick", 2 ], [ "2x4", 2 ] ],
```

Comment:
Plank (2x4) is fine, but sticks are insufficient for this, the canteen is about 6" in diameter and that face needs to be a single piece of wood, which you can't get from an average stick.